### ISSUE TITLE
make the automated boot2docker/boot2docker in the registry builds a bit more useful

### ISSUE THREAD

**Issue by AntonioMeireles** [boot2docker/boot2docker#755]
Hi, 

The point of those builds, as stated in the docs, is to add custom stuff to the *stock* ISO without having to go through the process of rebuilding everything, *ergo* it would be very, very, handy, IMHO to have tagged/named builds in the registry matching the boot2docker releases/ISOs, as, a priori, those are the ones one wants to match with custom addons. (one could start doing one now for v1.5.0 :smiley: )

**Comment by tianon** [comment]
Generally I agree, but this would mean we'd have to setup a new automated build for every tag we create (due to the way automated builds work).  If there were a way to tell the automated build system something like (for example) "make a Docker image tag of every Git tag that matches `v[0-9.]+`", I'd be 100% in support of including that.

**Comment by AntonioMeireles** [comment]
thnxs for your reply. i wonder if it wouldn't be possible to tag all future registry builds with the matching git(hub) commitId ? that would be enough for  this case afaict.

**Comment by SvenDowideit** [comment]
Un happily, automated build repositories are currently immutable - you can't tag or push to them (this is part of what is intended to make them safely related only to what is in GitHub

**Comment by AntonioMeireles** [comment]
humm... couldn't you create a side repo (simpler i suppose than some web service), say - *boot2docker-builds*- to where the automated builds machinery would '*post*' for each iteration, and on (docker) build success, something along **${githubCommitId} ${dockerHubBuildId}** ? just wondering.

**Comment by zfogg** [comment]
@AntonioMeireles that functionality could be created, but not as a github repo.. I believe it would require running some sort of web-connected application server.

**Comment by AntonioMeireles** [comment]
@zfogg :smile: i don't care about cat's color if they still catch mice. IMHO this kind of functionality would be very, very handy as it would allow turnkey docker/git plugable/tiered architectures.

**Comment by zfogg** [comment]
@AntonioMeireles agreed! I was just pointing out that it's probably outside the scope of what the b2d team wants to build

**Comment by AntonioMeireles** [comment]
ah, sure... so, in broader terms where would be the the proper 'channel' to *lobby* for this to get on the radar and implemented :smiley: ?

**Comment by zfogg** [comment]
hmm, good question! perhaps a docker IRC channel, or a docker mailing list? I'm sure many members of the docker community would be interested.

In my opinion and experience, the best way to get something built is to show initiative. Why not just start building it? Posting a working prototype in the IRC channel or the mailing list is much more likely to get interest and involvement; you'll have trouble starting the conversation with nothing other than the idea.
